### PR TITLE
Pr merge policies

### PR DESCRIPTION
**Pull request by chrisstafford**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201794020676141

### CODE REVIEW

**Review by joachimweyl — APPROVED** [review]
Looks good.

**Comment by larsks** [comment]
This isn't a policy so much as it is a "how to" document. I think we need to iterate on it a bit before merging.

**Review by larsks — CHANGES_REQUESTED** [review]
See my comment re: policy vs how-to

**Comment by joachimweyl** [comment]
@mikthoma do you feel that you can step in and make the changes necessary to format this as a policy rather than a How-to?

**Comment by larsks** [comment]
@joachimweyl I think the policy side might need to come from us.

**Comment by joachimweyl** [comment]
@mikthoma ignore that request.

**Comment by joachimweyl** [comment]
@jtriley do you have a set idea of PR merge policies that you could provide here?

**Review by jtriley — APPROVED** [review]

**Comment by jtriley** [comment]
This looks good as a contributing guide. I suppose a good initial policy for repos is enforcing protected branch for main, require ci/cd to pass, requiring two reviews, and all conversations to be resolved before merge.

**Comment by jtriley** [comment]
For CI/CD that deploys to a production system we should consider end user impact of the changes and schedule a maintenance event if necessary before merging.

**Comment by joachimweyl** [comment]
@larsks I have merged the policies from @chrisstafford's how-to and comments from @jtriley to create a policy list. Please let me know if this was more like what you had in mind.

File: pull_request_approval_policy.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Pull Request Approval Policies
+
+- Enforce protected branch for main
```
Comment (knikolla) [inline pull_request_approval_policy.md]: The first 3 lines aren't really approval policies for a PR as much as they are guidelines for setting up repos.

Comment (joachimweyl) [reply pull_request_approval_policy.md]: That makes sense. Thank you.
Updated.

File: pull_request_approval_policy.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Pull Request Approval Policies
+
+- CI/CD is required to pass
+- 2 Reviewers are required for all Pull Requests
+- All Change Requests must be resolved prior to merging
+- For CI/CD that deploy to a production system schedule a maintenance event if
```
Comment (larsks) [inline pull_request_approval_policy.md]: I think this last statement is a little wishy-washy.  How about:

- Changes that impact service availability must only be merged during a pre-scheduled maintenance window.

Comment (joachimweyl) [reply pull_request_approval_policy.md]: updated.

**Comment by joachimweyl** [comment]
This is the set of Policies for the creation of this project. NERC may choose to make changes to this once they are managing the PRs fully. At that time this doc might need to be updated.

**Review by larsks — APPROVED** [review]

**Pull request merged**